app/javascript/react/components/AnimalIndex.js

```diff
@@ -25,8 +25,8 @@ const AnimalIndex = (props) => {
 
   const animalList = animals.map((animal) => {
     return (
-      <div className="cell small-6 medium-4">
-        <AnimalTile key={animal.id} id={animal.id} name={animal.name} />
+      <div key={animal.id} className="cell small-6 medium-4">
+        <AnimalTile id={animal.id} name={animal.name} />
```

Comment:
You actually want this div to be inside of animal tile if you're going to add a key there. It's best practice for keys to be delivered inside a component.